Rocket.Chat/Managers/ServerManager.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-class ServerManager {
+struct ServerManager {
 
-    static let shared = ServerManager()
+    static var shared = ServerManager()
```

Comment:
Why var? 🤔

Reply:
Because I changed from class to be a struct (and it changes). 🤔